recipes/libglvnd/meta.yaml

```diff
@@ -0,0 +1,155 @@
+    sha256: d0e1925a3c9aee0143b8c181ac31d5637c8faa081759c277b8e16c7075612c11
+    patches:
+      - libglvnd_look_for_egl_icd_in_system_location.patch
+
+build:
+  skip: true  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - meson
+    - pkg-config
+    - ninja
+  host:
+    - xorg-libx11
+    - xorg-libxext
+    - xorg-xextproto
+    - xorg-glproto
+
+outputs:
+  - name: libglvnd
+    script: build_libglvnd.sh
+    build:
+      run_exports:
+        - {{ pin_subpackage('libglvnd', max_pin='x') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+        - meson
+        - pkg-config
+        - ninja
+      host:
+        - xorg-libx11
+        - xorg-libxext
+        - xorg-xextproto
+        - xorg-glproto
+    test:
+      commands:
+        - test -f $PREFIX/lib/libEGL.so.1
+        - test -f $PREFIX/lib/libGL.so.1
+        - test -f $PREFIX/lib/libGLESv1_CM.so.1
+        - test -f $PREFIX/lib/libGLESv2.so.2
+        - test -f $PREFIX/lib/libGLX.so.0
+        - test -f $PREFIX/lib/libOpenGL.so.0
+        - test -f $PREFIX/lib/libGLdispatch.so.0
+        - test ! -f $PREFIX/lib/pkgconfig/egl.pc
+        - test ! -f $PREFIX/lib/pkgconfig/gl.pc
+        - test ! -f $PREFIX/lib/pkgconfig/glesv1_cm.pc
+        - test ! -f $PREFIX/lib/pkgconfig/glesv2.pc
+        - test ! -f $PREFIX/lib/pkgconfig/glx.pc
+        - test ! -f $PREFIX/lib/pkgconfig/opengl.pc
+        - test ! -f $PREFIX/lib/pkgconfig/libglvnd.pc
+        - test ! -f $PREFIX/include/EGL/egl.h
+        - test ! -f $PREFIX/include/EGL/eglext.h
+        - test ! -f $PREFIX/include/EGL/eglplatform.h
+        - test ! -f $PREFIX/include/GL/gl.h
+        - test ! -f $PREFIX/include/GL/glcorearb.h
+        - test ! -f $PREFIX/include/GL/glext.h
+        - test ! -f $PREFIX/include/GL/glx.h
+        - test ! -f $PREFIX/include/GL/glxext.h
+        - test ! -f $PREFIX/include/GLES/egl.h
+        - test ! -f $PREFIX/include/GLES/gl.h
+        - test ! -f $PREFIX/include/GLES/glext.h
+        - test ! -f $PREFIX/include/GLES/glplatform.h
+        - test ! -f $PREFIX/include/GLES2/gl2.h
+        - test ! -f $PREFIX/include/GLES2/gl2ext.h
+        - test ! -f $PREFIX/include/GLES2/gl2platform.h
+        - test ! -f $PREFIX/include/GLES3/gl3.h
+        - test ! -f $PREFIX/include/GLES3/gl31.h
+        - test ! -f $PREFIX/include/GLES3/gl32.h
+        - test ! -f $PREFIX/include/GLES3/gl3ext.h
+        - test ! -f $PREFIX/include/GLES3/gl3platform.h
+        - test ! -f $PREFIX/include/KHR/khrplatform.h
+        - test ! -f $PREFIX/include/glvnd/GLdispatchABI.h
+        - test ! -f $PREFIX/include/glvnd/libeglabi.h
+        - test ! -f $PREFIX/include/glvnd/libglxabi.h
+  - name: libgl-devel
+    script: build_libgl_devel.sh
+    build:
+      run_exports:
+        - {{ pin_subpackage('libglvnd', max_pin='x') }}
```

Comment:
@hmaarrfk Are we sure this is supposed to be here for libgl-devel? A bit puzzled on the reason this exists...

Reply:
And why is there no option is `libgl-devel` to omit the `.so` libraries?

Reply:
Yes. This is correct. In fact the other run export should likely get removed. 

Let me try to write some docs that explain why

Reply:
> Yes. This is correct. In fact the other run export should likely get removed.

Could you make the changes you intended so that I understand the rationale better?

Reply:
@hmaarrfk 
Moreover, why does the libgl-devel recipe not contain `rm -rf "${PREFIX}/lib"`?
Doesn't that cause overlap in the files?

Reply:
at build time, it depends on the `libglvnd` so those files will automatically be found. So conda only detects "newly" files.

see https://github.com/conda-forge/conda-forge.github.io/pull/2257#issuecomment-2267578747 for run_exports justificaiton

Reply:
Yeah, makes sense.

Let's consider merging.